tests/__tests__/WinConditions.test.tsx

```diff
@@ -78,6 +79,17 @@ test('default win criteria', async () => {
     GameEnd { condition: null, conditionId: null, toPlayer: 1 }"
   `,
   );
+
+  const mapWithOptionalConditions = initialMap.copy({
+    config: initialMap.config.copy({
+      winConditions: initialMap.config.winConditions.map((c) => ({
+        ...c,
+        optional: true,
+      })),
```

Comment:
I know these are "just tests" but the code style is to make sure all variables are descriptive. Mind renaming `c` to `condition` everywhere?

Reply:
Completed in 991b485a951b0f88234d454a1fdfa617e7e5f7a2